### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2113

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/59031e7aaaace0c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/59031e7aaaace0c/output.txt

Total script time: 1.37 mins

Server is running on http://pr976.butyr.org/templates/basic/
Run the Test Suite: http://pr976.butyr.org/test/

File: src/layouts/trackevent.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
-  <div class="title"></div>
+  
+  <div class="butter-track-event-info">
+    <span class="butter-track-event-icon"></span>
+    <div class="title"></div>
+  </div>
+
   <div class="handle left-handle"></div>
   <div class="handle right-handle"></div>
+
```
Comment (secretrobotron) [inline src/layouts/trackevent.html]: Remove whitespace, please.

File: src/layouts/trackevent.html [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <div class="butter-track-event" data-butter-draggable-type="trackevent">
-  <span class="butter-track-event-icon"></span>
-  <div class="title"></div>
+
```
Comment (secretrobotron) [inline src/layouts/trackevent.html]: Remove whitespace, please (spaces on this line, not the line itself).

**Comment by mjschranz** [comment]
#984 is handling this now.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5b9b79ecbec7276/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/5b9b79ecbec7276/output.txt

Total script time: 3.00 mins

**Pull request closed without merging**